src/Schema/Events/UserExperienceInteractions.ts

```diff
@@ -0,0 +1,32 @@
+import { ContextModule } from "../Values/ContextModule"
+import { OwnerType } from "../Values/OwnerType"
+import { ActionType } from "."
+
+/**
+ * Schemas describing events for specific user experience design types.
+ * @packageDocumentation
+ */
+
+/**
+ * A user expands or collapses an accordion design, like in the consignments flow.
```

Comment:
**suggestion:** Should there be a way to differentiate between an expand and collapse action?

Reply:
I just added an `expand` boolean - thanks Adam!